sites/militaryaerospace.com/server/templates/index.marko

```diff
@@ -222,6 +222,14 @@ $ const { id, alias, name, pageNode } = data;
         </marko-web-query>
       </div>
     </div>
+    <div class="row">
+      <div class="col-lg-8 mb-block">
+        <leaders-home-page />
+      </div>
+      <div class="col-lg-4 mb-block">
+        <marko-web-gam-display-ad id="gpt-ad-rail2" />
```

Comment:
See the LFW leaders PR... a `gpt-ad-rail3` using the `rail1` ad unit needs to be created at the top this page. Otherwise this ad won't fire since `gpt-ad-rail2` is already defined and used previously.